frontend/src/components/UserSearch.vue

```diff
@@ -208,6 +255,45 @@ export default {
         parameters += "&" + parameter + "=" + value;
       }
       return parameters;
+    },
+    getClientsAndRoles: function() {
+      return ClientsRepository.get()
+        .then(response => {
+          this.clients = response.data;
+          this.getClientRoles();
+        })
+        .catch(e => {
+          console.log(e);
```

Comment:
We need some error handling here I think? Display the error the same as the others?